js/Heap.js

```diff
@@ -15,29 +16,32 @@ import Package from 'react-native-package';
  * https://github.com/negativetwelve/react-native-package
  */
 export default Package.create({
-  json: require('../package.json'),
+  json: require("../package.json"),
   nativeModule: NativeModules.RNHeap,
   enabled: Platform.select({
     ios: true,
+    android: true
   }),
-  export: (Heap) => ({
+  export: Heap => ({
     // App Properties
-    setAppId: (appId) => Heap.setAppId(appId),
+    setAppId: appId => Heap.setAppId(appId),
 
     // User Properties
-    identify: (identity) => Heap.identify(identity),
-    addUserProperties: (properties) => Heap.addUserProperties(properties),
+    identify: identity => Heap.identify(identity),
+    addUserProperties: properties => Heap.addUserProperties(properties),
 
     // Event Properties
-    addEventProperties: (properties) => Heap.addEventProperties(properties),
-    removeEventProperty: (property) => Heap.removeEventProperty(property),
+    addEventProperties: properties => Heap.addEventProperties(properties),
+    removeEventProperty: property => Heap.removeEventProperty(property),
     clearEventProperties: () => Heap.clearEventProperties(),
 
     // Events
     track: (event, payload) => Heap.track(event, payload),
 
-    // Config
-    enableVisualizer: () => Heap.enableVisualizer(),
-    changeInterval: (interval) => Heap.changeInterval(interval),
```

Comment:
I'm guessing these deletions are because android doesn't have these, but if it's possible maybe we could just make these a no-op on android instead?  I imagine some people might still find these methods useful, even if it's just on iOS.

Reply:
Yes, this is true.  Doing a NO-OP on Android was actually my first pass, but the more I looked at it, the more I didn't like it.  I could see keeping in the changeInterval (though the fact that Android doesn't have it and is a newer codebase might be telling) but I don't think the visualizer is going to be particularly useful in the RN world.